bin/pycbc_create_injections

```diff
@@ -238,7 +238,7 @@ while True:
         if old_samples is None:
             tstart = opts.gps_start_time
         else:
-            tstart = old_samples['tc'].max()
```

Comment:
I note that this does still create a discontinuity at the `draw_size` boundary. Now instead of the separations being *smaller* than normal, they will be *larger*. Does this matter? It might if you wanted a proper distribution.

I think it would be better to store the `last_time_step = tc[-1]` value at line 243, *before* you add the uniform bit, and then do `tstart = last_time_step + opts.time_step` here.

Reply:
I'm not sure I see why the separations are larger than expected now. Isn't just the start time of the block shifted by a fixed amount.

Reply:
Try it! Generate a set of injections with `draw_size` set to 10, and measure the distribution of spacing between subsequent injections for injections across, and not across, the boundary.

Reply:
@spxiwh @MarlinSchaefer I think larger is not a problem at the boundary since the main point is to maintain a minimum average separation, a maximum is not important beyond being able to fit enough signals in a set. 

@MarlinSchaefer I think your solution is OK.